crates/subspace-service/src/lib.rs

```diff
@@ -16,6 +16,9 @@
 
 //! Service and ServiceFactory implementation. Specialized wrapper over substrate service.
 
+pub mod chain_spec;
+pub mod rpc;
```

Comment:
Not sure PRC needs to be moved here either. It is a completely optional thing not needed for tests for instance.

Reply:
Polkadot has a separate crate for this rpc module, I didn't bother creating a new crate so just put it here. I can make it an argument of `new_full` if you prefer not always having this rpc in the service.

Reply:
Yes, I think for something like tests we definitely don't want to start HTTP/WS servers for no reason, it'll just make things more fragile and significantly slower to run.

Reply:
A flag to disable the rpc extensions added in https://github.com/subspace/subspace/pull/260/commits/7b6d0a41c190f32cd8ded13888b18d4a8582b4d7